README.md

```diff
@@ -51,10 +51,9 @@ and rav1e uses cargo (Rust). Check each library's documentation for an exact
 list of requirements.
 
 If you want to statically link any codec into your local (static) build of
-libavif, building using one of these scripts and then enabling the associated
-`AVIF_LOCAL_*` and setting `AVIF_CODEC_*` to `LOCAL` is a convenient method,
-but you must make sure to disable `BUILD_SHARED_LIBS` in CMake to instruct it
-to make a static libavif library.
+libavif, building using one of these scripts and then setting `AVIF_CODEC_*`
```

Comment:
Optional: Do you think it would be clearer if we add "the associated" after "setting", as in the original text?